docs/schema.md

```diff
@@ -678,6 +678,14 @@ The `affected` object's `versions` field is a JSON array of strings. Each string
 is a single affected version in whatever version syntax is used by the given
 package ecosystem.
 
+When there is no well-defined packaging ecosystem specified, (notably as for
```

Comment:
Hmm, there is a bit of an assumption here that it's generally accepted/understood that C/C++ libraries will only have GIT commit ranges. Would it make sense to instead say something like:

When there is no well-defined packaging ecosystem specified (for example, general C/C++ libraries), GIT commit ranges typically are the best way to define vulnerable version ranges. In this case, versions specified in this array cannot be relied upon to ...

Reply:
Reworded in b492a29